ontology/GALLONT.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: ontology_detail
+id: gallont
+title: "Plant Gall Ontology"
+contact:
+  email: adeans@psu.edu
+  github: adeans
+  label: Andy Deans
+  orcid: 0000-0002-2119-4663
+dependencies:
+  - id: pato
+  - id: po
+  - id: ro
+  - id: flopo
+  - id: caro
+  - id: obi
+  - id: poro
+  - id: ncbitaxon
+description: Ontology of plant gall phenotypes
```

Comment:
Please provide a more detailed description, outlining the purpose of the ontology and describing its content sufficiently so that users can assess, at a glance, its relevancy for their use cases.

Reply:
Will this work? "Plant galls are novel plant structures, generated by plants in response to biotic stressors. This ontology is used to annotate gall phenotypes -- e.g., their colors, textures, sizes, locations on the plant -- in a semantic way, in order to facilitate discoveries about the genetic and physiologic mechanisms responsible for such phenotypes. The ontology can also be used as a controlled vocabulary for natural language descriptions of plant galls."